.changeset/short-trainers-dance.md

```diff
@@ -0,0 +1,6 @@
+---
+"@nextui-org/progress": patch
```

Comment:
we don't need to add a changeset for stories changes. L2 can be removed.

Reply:
you're right! Done!